### PR TITLE
1.1.14 - rc

### PR DESCRIPTION
**Pull request by stevenweaver**

### CODE REVIEW

**Comment by rdvelazquez** [comment]
This is now live on staging.datamonkey.org. I'll try to review this soon so that we can get it merged into master and live on datamonkey.org.

File: test/loadBalTest.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 var assert = require('assert');
-var loadBalance = require('./../lib/loadBal.js');
+var loadBalance = require('./../lib/loadBal.js').loadBalance;
 var _ = require('underscore');
 
 //var urls = ["www.1.com", "www.2.com", "www.3.com", "www.4.com"]; //replace this with URL array
```
Comment (rdvelazquez) [inline test/loadBalTest.js]: Do we want this here?

**Comment by rdvelazquez** [comment]
We should also remove the twitter image at the bottom of the screen.

**Comment by stevenweaver** [comment]
Not compatible with HIV-TRACE. HIV-TRACE will need to be broken out to its own process that still makes use of `datamonkey 1.x.x`.

**Pull request merged**